### DISCUSSION TITLE
How to set the collision configuration of the robot so that each robot has its own collision configuration with Batched Environments

### DISCUSSION THREAD

**Discussion by yunding111** [NVlabs/curobo#175] [Software Q&A]
For example, there are multiple environments, each with the same robot and multiple different objects. I want to perform a pick-place task. After grabbing, each robot needs to attach the grabbed objects to ""attached_object" ", how to implement the plan using "Motion Generation"? Because I found that multiple robots in Batched Environments actually share a robot_cfg, so I cannot attach the capture of each environment to the robot instance of that environment.
Can anyone help me? Thanks

**Comment by balakumar-s** [comment]
This is not possible in the current implementation. See here for more details: https://github.com/NVlabs/curobo/discussions/91#discussioncomment-7930459
